modules/core/ShaderSSLT.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "ShaderFlat.h"
+#include "Scene.h"
+
+namespace rt {
+	/**
+		 * @brief Subsurface light transport shader
+		 */
+	class CShaderSSLT : public CShaderFlat {
+	private:
+		CScene& m_scene;
+		const Vec3f exitColor = RGB(0.4f, 0.4f, 0.4f);
+		const Vec3f	m_color = Vec3f::all(0.5f);
+
```

Comment:
I think it makes sense to add one more member-variable `m_opacity` here and initialize in the constructor as an argument